support-frontend/assets/pages/digital-subscription-landing/components/hero/hero.jsx

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+
+import { Button, buttonBrand } from '@guardian/src-button';
+import { SvgArrowDownStraight } from '@guardian/src-icons';
+import { type CountryGroupId, AUDCountries } from 'helpers/internationalisation/countryGroup';
+import GridImage from 'components/gridImage/gridImage';
+import {
+  wrapper,
+  pageTitle,
+  featureContainer,
+  textSection,
+  heroHeading,
+  yellowHeading,
+  paragraph,
+  heavyText,
+  packShot,
+  circle,
+  circleTextTop,
+  circleTextBottom,
+} from './heroStyles';
+
+type PropTypes = {
+  countryGroupId: CountryGroupId,
+}
+
+const HeroCopy = () => (
+  <span>
+    <p css={paragraph}>
+      With two innovative apps and ad-free reading, a digital subscription
+      gives you the richest experience of Guardian journalism. It also sustains the independent
+      reporting you love.
+    </p>
+    <p css={paragraph}>
+      For a few weeks only, read <span css={heavyText}>Edition Earth</span>, a new and exclusive
+      showcase of the best Guardian journalism on the climate, wildlife, air pollution, environmental
+      justice &mdash; and solutions.
+    </p>
+  </span>);
+
+const HeroCopyAus = () => (
+  <span>
+    <p css={paragraph}>
+      With two innovative apps and ad-free reading, a digital subscription gives you the richest experience
+      of Guardian Australia journalism. It also sustains the independent reporting you love.
+    </p>
+    <p css={paragraph}>
+      You&apos;ll gain exclusive access to <span css={heavyText}>Australia Weekend</span>, the new digital
+      edition, providing you with a curated view of the week&apos;s biggest stories, plus early access to
+      essential weekend news.
+    </p>
+  </span>);
+
+function CampaignHeader(props: PropTypes) {
+
+  return (
+    <div css={wrapper}>
+      <h1 css={pageTitle}>Digital subscription</h1>
+      <div css={featureContainer}>
+        <div css={textSection}>
+          <h2 css={heroHeading}>Progressive journalism<br />
+            <span css={yellowHeading}>powered by you</span>
+          </h2>
+          {props.countryGroupId === AUDCountries ? <HeroCopyAus /> : <HeroCopy />}
+          <ThemeProvider theme={buttonBrand}>
+            <Button
+              priority="tertiary"
+              size="default"
+              icon={<SvgArrowDownStraight />}
+              iconSide="right"
+              nudgeIcon
+              onClick={() => window.scrollTo(0, 1500)}
```

Comment:
Have you tested if this `y` coordinate 1500 works as expected across all breakpoints? If it doesn't not we may need to get the offsetTop of the element we want to scroll to and use that value instead.

Reply:
The `scrollIntoView` interface could do the job here too, and looks well supported: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Reply:
I have tested it across breakpoints and it's not exactly the same experience on all devices. From tablet upwards it appears in full in a nice, central position in the middle of the screen, which seems optimal. 

On phones it scrolls so the top of the pricing cards appears in about the middle of the screen:
![Screen Shot 2020-10-05 at 11 55 46](https://user-images.githubusercontent.com/16781258/95071775-1a72eb00-0702-11eb-8b02-6d01e501f8c5.png)

Do you think this is acceptable? Or should I adjust it?

Reply:
OK, I'd say that's totally acceptable for now, although we could revisit this to see if we can improve it

Reply:
👍